src/constants.js

```diff
@@ -10,9 +10,16 @@ export const DEFAULT_CRITERIA = {
 export const PICKUP_IN_STORE = 'pickup-in-point'
 export const DELIVERY = 'delivery'
 export const PICKUP = 'pickup'
+export const TAKE_AWAY = 'take-away'
 
 export const RESIDENTIAL = 'residential'
 export const COMMERCIAL = 'commercial'
 export const GIFT_REGISTRY = 'giftRegistry'
 
 export const SEARCH = 'search'
+
+export const SLA_TYPES = {
```

Comment:
Não seria melhor isso? Pq não são exatamente tipos de SLA e sim formas de entrega.

```suggestion
export const DELIVERY_CHANNELS = {
```